content/episode/002-testing.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Testing"
+episode: "2"
+Description: "Al and Andrew talk about testing. What is it, why is it useful, who should do it, etc..."
+podcast: "pdd-2.mp3"
+podcast_bytes: "27717141"
+podcast_duration: "00:43:58"
```

Comment:
Weirdly, this is the same length now :-\ but I don't care, it's just silence at the end. I'll figure that out for the next episode.